### PR TITLE
Add keyboard shortcuts to "toggle building" and "toggle building, then apply to same type"

### PR DESCRIPTION
**Pull request by sextondb**

### CODE REVIEW

**Comment by sextondb** [comment]
This has been split into two commits. One for the "toggle this building" and one for "toggle this building, then apply to buildings of the same type". The latter is a bit wordy, and I couldn't figure out a significantly easier way to phrase it that makes it clear that the similar buildings are set to the new value, and not toggled based on their current state/

**Comment by sextondb** [comment]
Useful for disabling production chains while still leaving them built. For example, it would make it easier to disable paper making and school, when changing focus to material production.

**Comment by sextondb** [comment]
Note that this is not a "pause/unpause" feature, as it always sets the capacity to 0% or 100%. It doesn't attempt to keep the previous values when toggling.

**Comment by sextondb** [comment]
Formatting issues fixed

**Comment by insraq** [comment]
Thanks. Do you want to be credited as `David Sexton` or `sextondb` in the patch notes?
